### PR TITLE
PROD-962 refactor(Change multiselect): Go multiselect style

### PR DESCRIPTION
**Pull request by MaaAx**

### CODE REVIEW

File: src/styles/gommette/components/_go-multiselect.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+$selected-icon-content: '\e933';
```
Comment (renanbronchart) [inline src/styles/gommette/components/_go-multiselect.scss]: You can put it in variables file

File: src/styles/gommette/components/_go-multiselect.scss [hunk 2]
```diff
@@ -53,7 +55,7 @@ fieldset[disabled] .multiselect {
 .multiselect__input,
 .multiselect__single {
   font-family: inherit;
-  font-size: 16px;
+  font-size: 14px;
   touch-action: manipulation;
```
Comment (renanbronchart) [inline src/styles/gommette/components/_go-multiselect.scss]: Is not you, but touch-action: manipulation is not available on Safari.

File: src/styles/gommette/components/_go-multiselect.scss [hunk 3]
```diff
@@ -103,27 +105,31 @@ fieldset[disabled] .multiselect {
...
+}
+
 .multiselect__input,
 .multiselect__single {
   position: relative;
   display: inline-block;
-  min-height: 20px;
-  line-height: 20px;
+  min-height: 21px;
+  line-height: 21px;
```
Comment (renanbronchart) [inline src/styles/gommette/components/_go-multiselect.scss]: height shouldn't be hardcoded. Look at type.sass file

File: src/styles/gommette/components/_go-multiselect.scss [hunk 4]
```diff
@@ -103,27 +105,31 @@ fieldset[disabled] .multiselect {
...
   border: none;
   border-radius: 3px;
   background: #fff;
-  padding: 0 0 0 5px;
+  padding: 0;
   width: calc(100%);
```
Comment (renanbronchart) [inline src/styles/gommette/components/_go-multiselect.scss]: Is not you but calc(100%). Is useless, just 100%

File: src/styles/gommette/components/_go-multiselect.scss [hunk 5]
```diff
@@ -103,27 +105,31 @@ fieldset[disabled] .multiselect {
...
 .multiselect__input,
 .multiselect__single {
   position: relative;
   display: inline-block;
-  min-height: 20px;
-  line-height: 20px;
+  min-height: 21px;
+  line-height: 21px;
   border: none;
   border-radius: 3px;
```
Comment (renanbronchart) [inline src/styles/gommette/components/_go-multiselect.scss]: The variable $border-radius-base exist

File: src/styles/gommette/components/_go-multiselect.scss [hunk 6]
```diff
@@ -162,7 +168,7 @@ fieldset[disabled] .multiselect {
 .multiselect__tag {
   position: relative;
   display: inline-block;
-  padding: 4px 26px 4px 10px;
+  padding: 3px 26px 3px 10px;
   border-radius: 3px;
```
Comment (renanbronchart) [inline src/styles/gommette/components/_go-multiselect.scss]: Same

File: src/styles/gommette/components/_go-multiselect.scss [hunk 7]
```diff
@@ -162,7 +168,7 @@ fieldset[disabled] .multiselect {
...
   margin-right: 10px;
```
Comment (renanbronchart) [inline src/styles/gommette/components/_go-multiselect.scss]: you can extend .m-b-sm

File: src/styles/gommette/components/_go-multiselect.scss [hunk 8]
```diff
@@ -252,10 +258,11 @@ fieldset[disabled] .multiselect {
   content: "";
 }
 
-.multiselect__placeholder {
+.multiselect__placeholder, 
+.multiselect__input::placeholder {
   color: #adadad;
   display: inline-block;
-  margin-bottom: 10px;
+  margin-bottom: 8px;
```
Comment (renanbronchart) [inline src/styles/gommette/components/_go-multiselect.scss]: Is it a magic number ?
Why 8 ?

File: src/styles/gommette/components/_go-multiselect.scss (hunk 1 above)
Comment (MaaAx) [reply src/styles/gommette/components/_go-multiselect.scss]: It's a variable, no?

File: src/styles/gommette/components/_go-multiselect.scss (hunk 2 above)
Comment (MaaAx) [reply src/styles/gommette/components/_go-multiselect.scss]: I don't know what it does on other browsers, do you think I should remove it?

Comment (MaaAx) [reply src/styles/gommette/components/_go-multiselect.scss]: I will let it as I don't know what it does exactly in our case. Do you agree?

**Pull request merged**